Rakefile

```diff
@@ -20,3 +20,15 @@ require 'hammer_cli_foreman_ansible/version'
 require 'hammer_cli_foreman_ansible/i18n'
 require 'hammer_cli/i18n/find_task'
 HammerCLI::I18n::FindTask.define(HammerCLIForemanAnsible::I18n::LocaleDomain.new, HammerCLIForemanAnsible.version.to_s)
+
+default_tasks = [:test]
```

Comment:
Another pattern you can use:
```ruby
task :default

begin
  require 'rubocop/rake_task'
rescue LoadError
  # RuboCop is optional
else
  RuboCop::RakeTask.new
  Rake::Task[:default].enhance([:rubocop])
end

Rake::Task[:default].enhance([:test])
```

Or:
```ruby
begin
  require 'rubocop/rake_task'
rescue LoadError
  task default: :test
else
  RuboCop::RakeTask.new
  task default: [:rubocop, :test]
end
```

It's too bad there's no way to prepend prequisites. IMHO you always want to run RuboCop before the test suite, which is why the code looks more complicated.

I'll leave it up to you which method you think is the most readable.

Reply:
Thanks! I prefer the second version you suggested.